### ISSUE TITLE
Orzly Nes mini controller to Wiimote on N64

### ISSUE THREAD

**Issue by Invictaz** [darthcloud/BlueRetro#416]
I connected a Wii Classic Pro (OEM) controller to the Wiimote. That works fine on the N64 via BR.

Then I disconnect the Wii Classic Pro controller and connect a 3rd party Orzly Nes mini controller to the Wiimote. It does nothing. 

The Wii Classic Pro controller has much more buttons and even analog sticks. 

Is this because it's a 3rd party controller or isn't it supported yet?

**Comment by darthcloud** [comment]
I guess they didn't bother implement the regular 6 bytes mode.

This build support the 8 bytes mode, but only if I can confirm it on after it's enable. Which they might have skip.

Anyway try this build:
[v1.8-rc2-4-g2f6c943_hw1_n64.zip](https://github.com/darthcloud/BlueRetro/files/9674924/v1.8-rc2-4-g2f6c943_hw1_n64.zip)

**Comment by Invictaz** [comment]
I cannot try the build as my RetroRosetta only has bluetooth OTA for the moment. Flashing the separate partition table is hard without a CP2102. The one I have might output 5V says GrechTech which can fry the controller ports he said.

Can you upload a firmware that is flashable via OTA?

**Comment by darthcloud** [comment]
all firmware are updatable via OTA, just use the bin file.

the other files are only for initial usb flash for a blank esp32.

**Comment by Invictaz** [comment]
@darthcloud with your firmware the fake NES mini controller plugged into the Wiimote started working. However, I have very weird stick drift in Perfect Dark. The character walks around in circles. And the button mapping is weird.

I tried doing live output over bluetooth but that doesnt seem to work on the Retrorosetta. Also not on the older 1.7.3 firmware.

**Comment by darthcloud** [comment]
I think it required the 8 bytes mode config to start working but then probably report itself to be still in 6 bytes mode which would explain what you see, ie a 8bytes frame decoded as 6 bytes.

I'll send you another build to test later.

**Comment by darthcloud** [comment]
try this
[v1.8-rc2-5-gca5d95a_hw1_n64.zip](https://github.com/darthcloud/BlueRetro/files/9677843/v1.8-rc2-5-gca5d95a_hw1_n64.zip)

**Comment by Invictaz** [comment]
Thanks for the fast update. Some good and bad news: 

Found out that the fake NES mini controller I have mostly looks like the 3rd party Orzly branded controller.

The stick drift after your update is still there. However, it can be mitigated partially: reconnect the classic pro controller again to the Wiimote and press sync on the WiiMote and then swap back the Orzly nes controller = drift stops. But not in Perfect Dark. I have only tested this succesfully in Mario 64. Perfect Dark still drifts. 

So if this firmware did change anything I'm not sure.
